app/assets/stylesheets/lifehack_style.scss

```diff
@@ -160,9 +188,70 @@
 }
 
 .add-lifehack {
-  border: 1px solid $green;
+  border: 0.1rem solid $oil;
   border-radius: 28px;
-  display: inline-block;
   font-size: 15px;
-  padding: 12px 20px;
+  width: 500px;
+  font-size: 26px;
+  background-color: $turquoise;
+}
+
+.top-action-bar {
+  text-align: center;
+  padding-bottom: 15px;
+}
+
+.review {
+
+  table {
+    border-bottom: 2px solid $oil;
+    margin-top: 20px;
+    background-color: $cornsilk;
+  }
+  th {
+    text-align: center;
+    padding:1px 71px 0px 70px;
+  }
+
+  td {
+    text-align: center;
+      padding: 0 0 0 0;
+  }
+
+  .lifehack-link a {
+    padding: 0 0 40px 0;
+  }
+
+  a {
+    color: $navy-blue;
+    font-weight: bold;
+  }
+
+  .review-admin-panel a {
+    border: 1px solid $oil;
+    color: $primary-color;
```

Comment:
Properties should be ordered border, border-radius, color, display, font-size, padding